pubspec.yaml

```diff
@@ -59,8 +60,7 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+assets:
```

Comment:
Because in the beggining i wanted to use the images for the backdrop and the poster as assets, and then i changed my mind and used them from the network instead. I forgot to comment it back! Fixed it :)